src/main/deliferoo/xml/XMLParser.java

```diff
@@ -21,6 +21,13 @@
 import javax.xml.stream.XMLStreamException;
 import javax.xml.stream.XMLStreamReader;
 
+/**
+ * Class of singleton extracting datas from given 
+ * XML file by parsing them.
+ * 
+ * @authors hakima, marie, sadsitha, louis, teck
+ */
+
```

Comment:
Mets plutôt :
"Class for parsing and extracting data from a given XML file
Implements singleton instantialisation" :)